src/server.cpp

```diff
@@ -957,17 +957,19 @@ Server::~Server()
 	}
 }
 
-void Server::start(unsigned short port)
+void Server::start(Address bind_addr)
```

Comment:
Er, maybe?  This pull is very old though, that would be a separate patch.  You can fix it yourself if it doesn't break anything.